me.sergiotarxz.JapaChar.yml

```diff
@@ -0,0 +1,99 @@
+id: "me.sergiotarxz.JapaChar"
```

Comment:
`sergiotarxz.me` is 404 not found.

Reply:
It is no longer 404. I thought only by having the domain was enough.

Reply:
and how would we verify otherwise?